calibrate/README.md

```diff
@@ -7,6 +7,20 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 - This project requires [Node version 14](https://nodejs.org/dist/) and
 - [NPM](https://docs.npmjs.com/)
 
+## Prerequisites
+Use the [.env.sample](./.env.sample) to create a personal `.env` file. 
+
+- `.env` file
+```
+PORT=5000
+NODE_PATH=src/
+REACT_APP_BASE_URL=http://staging-platform.airqo.net/api/v1/
+REACT_APP_DOCS_BASE_URL=https://staging-docs.airqo.net/#/
+REACT_APP_BASE_CALIBRATE_URL=
+REACT_APP_AUTHORIZATION_TOKEN=ey123abc
+```
```

Comment:
Could we remove this part since it's already included in the `.env.sample` file

Reply:
Hi @123MwanjeMike.
I just added another commit that makes this change. Please check it out 😄